scripts/examples/50-OpenMV-Boards/52-Alif-Boards/51-OLED-Shield/oled_display_with_joystick.py

```diff
@@ -0,0 +1,71 @@
+# This work is licensed under the MIT license.
+# Copyright (c) 2013-2025 OpenMV LLC. All rights reserved.
+# https://github.com/openmv/openmv/blob/master/LICENSE
+#
+# OLED Display with Joystick Example
+#
+# Note: To run this example you will need a OLED Breakout Board for your OpenMV AE3
+#
+# The OLED Breakout Board allows you to view your OpenMV AE3's frame buffer on the go.
+
+import sensor
+import time
+import display
+from machine import I2C
+from machine import Pin
+
+I2C_EXPANDER_ADDR = 63
+
+sensor.reset()
+sensor.set_pixformat(sensor.RGB565)
+sensor.set_framesize(sensor.VGA)
+sensor.set_windowing((400, 400))
+
+i2c = I2C(1)
+exp_int_pin = Pin("P9", Pin.IN)
+lcd = display.SPIDisplay(width=128, height=128, controller=display.SSD1351())
+clock = time.clock()
+
+
+def expander_has_event():
+    return exp_int_pin.value() ^ 1
+
+
+def read_expander():
+    data = i2c.readfrom(I2C_EXPANDER_ADDR, 1)[0] ^ 255
+    right = (data & 1)
+    up = ((data >> 1) & 1)
+    left = ((data >> 2) & 1)
+    down = ((data >> 3) & 1)
+    center = ((data >> 4) & 1)
+    lcd_on = ((data >> 5) & 1)
+    return right, up, left, down, center, lcd_on
```

Comment:
This should be a frozen module and it should use a real IRQ pin. Best to remove this for now if not ready.

Reply:
Dropping example. Can you take on adding the pin irq to alif and getting it merged upstream?